search/mindbender-search

```diff
@@ -24,6 +24,13 @@
 ##
 # Author: Jaeho Shin <netj@cs.stanford.edu>
 # Created: 2015-07-31
+
+##Remark: the default heap size of Elastic search (1GB) may be insufficient for the application, resulting in error "Data too large, data for [type] would be larger than limit of [622775500/593.9mb]".
+# To modifiy it, run:
+# export ES_HEAP_SIZE=10g
+# before mindbender search gui. Pick a value adapted for your machine but lower than 30.5GB. More info: https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html
```

Comment:
Yes, it happens during search gui, the error is actually displayed by elastic search when we open it in a browser.
I was loading 54GB of data split into 5 tables (30GB, 14GB, 9GB, 0.5GB, 0.5GB). I increased the ES_HEAP_SIZE to 10GB and the error disappeared and the views worked correctly.